flink-sql-runner/src/main/java/com/github/streamshub/flink/SqlRunner.java

```diff
@@ -74,44 +84,101 @@ public static void main(String[] args) throws Exception {
         }
     }
 
-    static List<String> parseStatements(String rawStatements) {
-        var formatted = formatSqlStatements(rawStatements.trim());
+    static List<String> parseStatementArg(String rawStatementArg) {
```

Comment:
Why re-name this method? `parseStatement` still feels like the right name.

Reply:
Well now we are parsing all the args, which each could be just one statement or multiple, it felt better to call a spade a spade.